django-backend/fecfiler/web_services/dot_fec/test_dot_fec_serializer.py

```diff
@@ -78,19 +85,34 @@ def test_serialize_field(self):
         )
         self.assertEqual(serialzed_date_undefined, "")
 
-        # BOOLEAN
-        serialized_boolean_true = serialize_field(
+        # BOOLEAN_X
+        serialized_boolean_x_true = serialize_field(
             self.f3x, "change_of_address", f3x_field_mappings
         )
-        self.assertEqual(serialized_boolean_true, "X")
-        serialized_boolean_false = serialize_field(
+        self.assertEqual(serialized_boolean_x_true, "X")
+        serialized_boolean_x_false = serialize_field(
             self.f3x, "qualified_committee", f3x_field_mappings
         )
-        self.assertEqual(serialized_boolean_false, "")
-        serialized_boolean_undefined = serialize_field(
+        self.assertEqual(serialized_boolean_x_false, "")
+        serialized_boolean_x_undefined = serialize_field(
             F3XSummary(), "qualified_committee", f3x_field_mappings
         )
-        self.assertEqual(serialized_boolean_undefined, "")
+        self.assertEqual(serialized_boolean_x_undefined, "")
+
+        # BOOLEAN_YN
+        serialized_boolean_yn_true = serialize_field(
+            self.schc_transaction1, "secured", schc_field_mappings
+        )
+        self.assertEqual(serialized_boolean_yn_true, "Y")
+        serialized_boolean_yn_false = serialize_field(
+            self.schc_transaction1, "personal_funds", schc_field_mappings
+        )
+        self.assertEqual(serialized_boolean_yn_false, "N")
+        serialized_boolean_yn_undefined = serialize_field(
+            self.schc_transaction2, "personal_funds", schc_field_mappings
+        )
+        # N Because model has default=False, otherwise would be ''
+        self.assertEqual(serialized_boolean_yn_undefined, "N")
```

Comment:
You know this sheds light on that setting.  I used the same field config from the transaction model when making the schedule c model.  But for Y/N, we probably want to know whether the field/question has been answered yet.